far/FarEng.hlf.m4

```diff
@@ -2261,6 +2261,9 @@ executing command. <title> and <init> - title and initial text of edit control.
 
  Leave the name empty to disable history.
 
+ If the title field ends with a #|# the title itself is used internally as the history key for the input text (instead of #UserVarN#)
```

Comment:
It's a neat idea. However, have you seen the previous paragraph?

Reply:
Yes, but I couldn't figure out what `name` is referring to.
I 've tried `notepad.exe !??!`, but it keeps both, history and last input text.

Reply:
[`!?$<history>$<title>?<init>!`](https://github.com/FarGroup/FarManager/blob/13d47a573f87c8a2a3d5bf4d1e9ae07e98ed80b2/far/FarEng.hlf.m4#L2258-L2260)

Reply:
Nice, closing the PR